test/lic5Test.java

```diff
@@ -0,0 +1,59 @@
+import decide.Connectors;
+import decide.Coordinate;
+import decide.Decider;
+import decide.Parameters;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class lic5Test {
+
+    @Test
+    void lic5_test0() {
```

Comment:
Test names include no description of what is tested, which makes them difficult to read/understand.